### ISSUE TITLE
Basemap: note that WGS84 is not a projection, so the error message needs to be edited.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#188]
```
Basemap: note that WGS84 is not a projection, so the error message needs to be 
edited.
```

Original issue reported on code.google.com by `lixun...@gmail.com` on 30 Jun 2015 at 7:33

**Comment by jkoschinsky** [comment]
The feedback I'm getting on the current text is that this doesn't tell people how to specifically fix the problem. Let's update to this text:

To add the base map, you need a .prj file (WGS84 format) in the same directory as your spatial files.

**Comment by jkoschinsky** [comment]
works for Luc

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.85 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).
